console_bot/address_book.py

```diff
@@ -234,6 +234,48 @@ def get_next_week_birthdays(self):
 
         return get_birthdays_per_week(result)
 
+    def add_email(self, name, email):
+        if name in self.data:
+            is_valid, message = self.data[name].add_email(email)
+            return message
+        else:
+            return f"Contact {name} not found."
+
+    def change_email(self, name, new_email):
+        if name in self.data:
+            self.data[name].emails.clear()  # Assuming each contact has only one email for simplicity
+            is_valid, message = self.data[name].add_email(new_email)
+            return message
+        else:
+            return f"Contact {name} not found."
+
+    def get_email(self, name):
+        if name in self.data and self.data[name].emails:
+            return ', '.join(email.value for email in self.data[name].emails)
+        else:
+            return f"Email for contact {name} not found or not set."
+
+    def add_address(self, name, address):
+        if name in self.data:
+            is_valid, message = self.data[name].add_address(address)
+            return message
+        else:
+            return f"Contact {name} not found."
+
+    def change_address(self, name, new_address):
+        if name in self.data:
```

Comment:
Technically change works as `add` if you call it on contact with empty address, which is fine